### PR TITLE
Update FritzBox call monitor documentation to match HA PR #6474

### PR DESCRIPTION
**Pull request by DavidMStraub**
**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#6474

### CODE REVIEW

**Comment by Skorfulose** [comment]
Hi David, cool new feature. The doc changes seem good besides one thing: Can you add a hint on the permissions the FRITZ!Box user actually needs to be able to fetch the phone book? I'd think creating a new user with limited permissions would be best practice (over using the full admin user).

**Comment by DavidMStraub** [comment]
Good point - I added a sentence.

File: source/_components/sensor.fritzbox_callmonitor.markdown [hunk 1]
```diff
@@ -2,7 +2,7 @@
 layout: page
 title: "FRITZ!Box"
 description: "Instructions how to integrate a phone call monitor for AVM FRITZ!Box routers into Home Assistant."
-date: 2016-08-13 15:00
+date: 2017-03-07 22:00
```
Comment (Landrash) [inline source/_components/sensor.fritzbox_callmonitor.markdown]: Don't change this date. The creation of this page has not changed.

File: source/_components/sensor.fritzbox_callmonitor.markdown [hunk 2]
```diff
@@ -31,3 +32,29 @@ Configuration variables:
...
+- **username** (*Optional*): Fritz!Box user's user name. This is required to use the phone book lookup feature. The user needs to have the "voice message, fax message, Fritz!App Fon and call list" permission.
+- **password** (*Optional*): Fritz!Box user's user password. This is required to use the phone book lookup feature.
+- **phonebook** (*Optional*): Numerical ID identifying the phonebook to be used. If there is just one phonebook, this is usually 0.
+- **prefixes** (*Optional*): In case of a local call, the phone number seen by the router might differ from the one stored in the phone book by an area code, similarly for the international prefix. To remedy this, a list of prefixes, that can be appended to the phone number in case it is not found in the phone book, can be given.
+
+## {% linkable_title Examples %}
+
+In this section you find some real life examples of how to use this lock.
+
+### {% linkable_title Full configuration %}
```
Comment (Landrash) [inline source/_components/sensor.fritzbox_callmonitor.markdown]: A minimal config example is missing.

Comment (DavidMStraub) [inline source/_components/sensor.fritzbox_callmonitor.markdown]: The minimal example was already present in the old version.

File: source/_components/sensor.fritzbox_callmonitor.markdown (hunk 1 above)
Comment (DavidMStraub) [inline source/_components/sensor.fritzbox_callmonitor.markdown]: Reverted in 38188de34f96493fff5e1f2a38f7bbf47cfd0eef.

File: source/_components/sensor.fritzbox_callmonitor.markdown (hunk 2 above)
Comment (Landrash) [inline source/_components/sensor.fritzbox_callmonitor.markdown]: Your correct, had the wrong window up for that comment. Sorry, my mistake.

File: source/_components/sensor.fritzbox_callmonitor.markdown [hunk 3]
```diff
@@ -31,3 +32,29 @@ Configuration variables:
...
 - **host** (*Optional*): The IP address of your router, eg. 192.168.1.1. It is optional since every fritzbox is also reachable by using the IP address 169.254.1.1.
 - **port** (*Optional*): The TCP port of the call monitor. There is usually no reason to change this.
+- **username** (*Optional*): Fritz!Box user's user name. This is required to use the phone book lookup feature. The user needs to have the "voice message, fax message, Fritz!App Fon and call list" permission.
+- **password** (*Optional*): Fritz!Box user's user password. This is required to use the phone book lookup feature.
+- **phonebook** (*Optional*): Numerical ID identifying the phonebook to be used. If there is just one phonebook, this is usually 0.
+- **prefixes** (*Optional*): In case of a local call, the phone number seen by the router might differ from the one stored in the phone book by an area code, similarly for the international prefix. To remedy this, a list of prefixes, that can be appended to the phone number in case it is not found in the phone book, can be given.
+
+## {% linkable_title Examples %}
+
+In this section you find some real life examples of how to use this lock.
```
Comment (Landrash) [inline source/_components/sensor.fritzbox_callmonitor.markdown]: Fairly certain this isn't a lock?

Comment (DavidMStraub) [inline source/_components/sensor.fritzbox_callmonitor.markdown]: Oops ;) removed the sentence.  Thanks for spotting it.

**Comment by DavidMStraub** [comment]
Hi, FYI the [PR to HA](https://github.com/home-assistant/home-assistant/pull/6474) has just been merged.

**Pull request merged**